packages/twenty-front/src/modules/workflow/workflow-diagram/components/WorkflowDiagramStepNodeBase.tsx

```diff
@@ -173,6 +180,7 @@ export const WorkflowDiagramStepNodeBase = ({
           type="target"
           position={Position.Top}
           isVisible={displayHandle}
+          selected={handlesSelectedState.topHandle}
```

Comment:
Reactflow differentiates `target` and `source` handles. A node can have several target or source handles. Maybe we should keep the same terminology and have:

```ts
handlesSelectedState.defaultTargetHandle
handlesSelectedState.defaultSourceHandle
```

Or, as it would be enough for the foreseeable future:

```ts
handlesSelectedState.targetHandle
handlesSelectedState.sourceHandle
```